<ui version="4.0" >
 <class>HelpBrowser2</class>
 <widget class="QDialog" name="HelpBrowser2" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible" >
      <bool>false</bool>
     </property>
     <widget class="QTabWidget" name="tabWidget" >
      <property name="currentIndex" >
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabContents" >
       <attribute name="title" >
        <string>&amp;Contents</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QTreeWidget" name="listView" >
          <property name="verticalScrollBarPolicy" >
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <column>
           <property name="text" >
            <string>1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSearching" >
       <attribute name="title" >
        <string>Se&amp;arch</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLineEdit" name="searchingEdit" >
            <property name="toolTip" >
             <string>Searching is case insensitive</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="searchingButton" >
            <property name="text" >
             <string>&amp;Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="searchingView" />
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabBookmarks" >
       <attribute name="title" >
        <string>Book&amp;marks</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QTreeWidget" name="bookmarksView" >
          <column>
           <property name="text" >
            <string>1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTextEdit" name="textBrowser" />
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
